File: Java/opentraceability/src/main/java/opentraceability/mappers/EPCISQueryDocumentMappers.java
package opentraceability.mappers;


import opentraceability.interfaces.*;
import opentraceability.mappers.epcis.json.*;
import opentraceability.mappers.epcis.xml.EPCISQueryDocumentXMLMapper;
import opentraceability.mappers.masterdata.*;
import opentraceability.models.events.*;
import opentraceability.*;

public class EPCISQueryDocumentMappers
{
	public IEPCISQueryDocumentMapper XML = new EPCISQueryDocumentXMLMapper();
	public IEPCISQueryDocumentMapper JSON = new EPCISQueryDocumentJsonMapper();
}

File: Java/OpenTraceability/src/main/opentraceability/mappers/epcisquerydocumentmappers.kt
package mappers

import interfaces.IEPCISQueryDocumentMapper
import mappers.epcis.json.EPCISQueryDocumentJsonMapper
import mappers.epcis.xml.EPCISQueryDocumentXMLMapper

class EPCISQueryDocumentMappers {
    var XML: IEPCISQueryDocumentMapper = EPCISQueryDocumentXMLMapper()
    var JSON: IEPCISQueryDocumentMapper = EPCISQueryDocumentJsonMapper()
}
